pixl_imaging/tests/docker-compose.yml

```diff
@@ -54,6 +54,7 @@ services:
       - "9001:8000"
 
   vna-qr:
+    platform: linux/amd64
```

Comment:
are these all needed?

Reply:
Having looked into it the answer appears to be yes. Also need to set `DOCKER_DEFAULT_PLATFORM=linux/arm64` for it to work locally which I'm not sure why.

Error when no platform is specified:
`Error response from daemon: image with reference osimis/orthanc:22.9.0-full-stable was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64 `

Error if I set `DOCKER_DEFAULT_PLATFORM=linux/amd64` (both with and without platform specified):
`Error response from daemon: image with reference rabbitmq:3.12.9-management was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64`

Reply:
Same pattern of errors for the system tests

Reply:
I think you'll need to down all of your containers and then do `docker system prune --all` as its downloaded that version on arm and doesn't seem to want to redownload it

Reply:
Yeah, I don't think all that is necessary, I'm able to run these containers fine on my M2 (would surprise me if there would be a difference between M1-M2 in this case).
Make sure you have Rosetta enabled as well in Docker settings.
<img width="1382" alt="Screenshot 2024-02-09 at 09 52 17" src="https://github.com/UCLH-Foundry/PIXL/assets/38256462/79cd6dd0-ed8f-4a1a-bb8e-486c12868949">